azure/sql-server/README.md

```diff
@@ -0,0 +1,242 @@
+
+        - (number) **`vcore`** _[since v0.0.1]_
+
+            Defines the number of VCores for the database. Please run the above command to get a list of VCores options applicable to your region.
+
+        - (number) **`auto_pause_delay_in_minutes = -1`** _[since v0.0.1]_
+
+            Time in minutes after which database is automatically paused. A value of `-1` means that automatic pause is disabled. This property is only applicable for `Serverless` tier
+
+        - (string) **`compute = "Gen5"`** _[since v0.0.1]_
+
+            Defines the compute for the database. Note that certain compute options are only available to certain tiers, and may not be available in some regions. Run this CLI command to get a list of options applicable to your region. `az sql db list-editions --location westus --output table`
+
+        - (number) **`min_vcore = 1`** _[since v0.0.1]_
+
+            Minimal capacity that database will always have allocated, if not paused. This property is only applicable for `Serverless` tier.
+
+    - (bool) **`ledger_enabled = false`** _[since v0.0.1]_
+
+        Specifies if this is a ledger database, cannot be changed after database creation
+
+    - (bool) **`read_scale_out_enabled = true`** _[since v0.0.1]_
+
+        If enabled, connections that have application intent set to readonly in their connection string may be routed to a readonly secondary replica. This property is only settable for `Premium` and `BusinessCritical` tiers.
+
+    - (string) **`restore_point_in_time = null`** _[since v0.0.1]_
+
+        Specifies the point in time (ISO8601 format) of the source database that will be restored to create the new database. This property is only settable for `create_mode = "PointInTimeRestore"` databases.
+
+    - (string) **`source_database_id = null`** _[since v0.0.1]_
+
+        The ID of the source database from which to create the new database. This should only be used for databases with create_mode values that use another database as reference. Changing this forces a new resource to be created.
+
+    - (bool) **`zone_redundant = false`** _[since v0.0.1]_
+
+        Whether or not this database is zone redundant, which means the replicas of this database will be spread across multiple availability zones. This property is only settable for `Premium` and `BusinessCritical` tiers.
+
+- (object) **`firewall = null`** _[since v0.0.1]_
+
+    Manages firewall rules to allow incoming traffic
+
+    - (map(string)) **`rules`** _[since v0.0.1]_
+
+        A map of firewall rules in the following format `{rule_name = start_ip - end_ip}`. For example. `"{Office's Network" = "1.2.3.4 - 5.6.7.8"}`. If `start_ip` and `end_ip` are identical, you can omit `end_ip`. For example. `"{Peter's home network" = "1.2.3.4"}`
```

Comment:
```suggestion
        A map of firewall rules in the following format: `{"rule_name" = "start_ip - end_ip"}`. For example. `{"Office's Network" = "1.2.3.4 - 5.6.7.8"}`. If `start_ip` and `end_ip` are identical, you can omit `end_ip`. For example. `{"Peter's home network" = "1.2.3.4"}`
```

I think this is what you mean in your examples?

Reply:
good catch. fixed